composer/modules/server/services/ballerina-parser/src/main/java/org/ballerinalang/composer/service/ballerina/parser/service/util/BLangFragmentParser.java

```diff
@@ -137,15 +137,16 @@ protected static JsonObject getJsonNodeForFragment(JsonObject jsonModel, BLangSo
     }
 
     protected static JsonElement getJsonModel(WorkspaceDocumentManager documentManager, String source)
-            throws IOException, InvocationTargetException, IllegalAccessException {
-        BLangCompilationUnit compilationUnit = null;
-        java.nio.file.Path filePath = LSCompiler.createAndGetTempFile(UNTITLED_BAL);
-        BallerinaFile model = LSCompiler.compileContent(source, filePath, CompilerPhase.DEFINE, documentManager, true);
-        if (model.getBLangPackage() != null) {
-            compilationUnit = model.getBLangPackage().getCompilationUnits().stream().
-                    filter(compUnit -> UNTITLED_BAL.equals(compUnit.getName())).findFirst().get();
-        }
-        return TextDocumentFormatUtil.generateJSON(compilationUnit, new HashMap<>());
+            throws LSCompilerException, JSONGenerationException {
+        java.nio.file.Path filePath = LSCompilerUtil.createTempFile(LSCompilerUtil.UNTITLED_BAL);
```

Comment:
Any specific reason to add the full package path here?

Reply:
Nope, fixed with https://github.com/ballerina-platform/ballerina-lang/pull/9836/commits/b578fda8ac13fe77028c276521a36f6be7a2067b